File: tests/testdata/codegen/transpiled_examples/getting-started-pp/python/getting-started.py
import khulnasoft
import khulnasoft_aws as aws

mybucket = aws.s3.Bucket("mybucket", website={
    "index_document": "index.html",
})
indexhtml = aws.s3.BucketObject("indexhtml",
    bucket=mybucket.id,
    source=khulnasoft.StringAsset("<h1>Hello, world!</h1>"),
    acl="public-read",
    content_type="text/html")
khulnasoft.export("bucketEndpoint", mybucket.website_endpoint.apply(lambda website_endpoint: f"http://{website_endpoint}"))

File: tests/testdata/codegen/transpiled_examples/getting-started-pp/nodejs/getting-started.ts
import * as khulnasoft from "@khulnasoft/khulnasoft";
import * as aws from "@khulnasoft/aws";

const mybucket = new aws.s3.Bucket("mybucket", {website: {
    indexDocument: "index.html",
}});
const indexhtml = new aws.s3.BucketObject("indexhtml", {
    bucket: mybucket.id,
    source: new khulnasoft.asset.StringAsset("<h1>Hello, world!</h1>"),
    acl: "public-read",
    contentType: "text/html",
});
export const bucketEndpoint = khulnasoft.interpolate`http://${mybucket.websiteEndpoint}`;
